l10n_ch_lsv_dd/model/banking_export_ch_dd.py

```diff
@@ -0,0 +1,95 @@
+##############################################################################
+#
+#    Swiss localization Direct Debit module for OpenERP
+#    Copyright (C) 2014 Compassion (http://www.compassion.ch)
+#    @author: Cyril Sester <cyril.sester@outlook.com>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models, fields, api, _
+from openerp.addons.decimal_precision import decimal_precision as dp
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+class banking_export_ch_dd(models.Model):
+
+    ''' Swiss Direct Debit export containing the file created
+        by the appropriate wizard
+    '''
+    _name = 'banking.export.ch.dd'
+    _rec_name = 'filename'
+
+    def _generate_filename(self):
+        self.ensure_one()
+        ref = self.env['ir.sequence'].next_by_code(
+            'l10n.banking.export.filename')
+        username = self.env.user.name
+        initials = ''.join([subname[0] for subname in username.split()])

```

Comment:
Ok let's keep it that way. However it would be nice to have a field on user to define the initials, making it possible to have uniq initials in case you have a John Doe and a Joe Dalton.